root/etc/systemd/system/setup-extension.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Download and enable zip extension to this image
+Wants=basic.target
+After=basic.target network.target network-online.target nss-lookup.target
+
+[Service]
+Environment=LD_LIBRARY_PATH=/opt/extension/lib
+Environment=RUBYLIB=/opt/extension/lib/ruby
+Environment=PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/opt/extension/bin
```

Comment:
At the very least we can write our own EnvironmentFile and reference it here. I'll fix it.